### DISCUSSION TITLE
Migrating from Paperless addon to the new recommended addon?

### DISCUSSION THREAD

**Discussion by Valve90210** [alexbelgium/hassio-addons#1293] [General]
Hi

I've been using your paperless-ngx addon for a while and love it but i see you're recommending a different add on now which I will move to. But I have no idea how I can migrate my documents, tags etc? And as far as I can see there doesn't appear to be a simple way to backup the database on the old addon and restore it to the new?

Are you able to offer any guidance by any chance?

**Comment by eikeja** [comment]
I have the same problem. I am completely lost and have no idea how best to solve the problem.

If anyone here could provide some guidance, that would be great. I would also donate something to the person.

I can't possibly reassign all the tags, correspondents and settings.

**Comment by eikeja** [comment]
I managed to import all my data into the new BenoitAnastay add-on!

It was actually pretty easy.

- Install the new addon
- Make a backup of the old Paperless directory. In my case "/addon_configs/db21ed7f_paperless_ng"
- View the folder structure of the new instance, assign files from the old instance to the folders of the new instance.
- Start new Paperless - all data is there!

**Folder assignment:**
/addon_configs/db21ed7f_paperless_ng/data → /addon_configs/ca5234a0_paperless-ngx/data

/addon_configs/db21ed7f_paperless_ng/media → /share/paperless

Done!

**Comment by alexbelgium** [comment]
Thanks very much! I'll add your way of working to the deprecation notice

**Comment by KapUttyy** [comment]
What is meant by assign files from old instance to the folders of the new instance?
Do you mean copy those files into the new instance?

My Paperless NGX instance by @alexbelgium is running for about a year now and is linked to the MariaDB backend. 

As far as I can see, I am not able to configure the Database in the Add-on by BenoitAnastay. Is there a way to migrate from MariaDB to SQLite?

**Comment by eikeja** [comment]
Sorry, I wrote that stupidly. It is said that the data must be copied from folder A to folder B, yes!

I can't say how it works with MariaDB. I haven't changed anything in any database and I know very little about it 😔

**Reply by KapUttyy** [reply]
@alexbelgium do you have any idea how to migrate the database correctly?
If not I would like to try using the exporter which is included by paperless ngx: https://docs.paperless-ngx.com/administration/#exporter but when I try to run those commands with the Advanced SSH & Web Terminal Add-On, I just get errors. After that I'd like to import this to another docker host.
